Kernel/Interrupts/IRQHandler.h

```diff
@@ -32,6 +32,7 @@ class IRQHandler : public GenericInterruptHandler {
     virtual size_t sharing_devices_count() const override { return 0; }
     virtual bool is_shared_handler() const override { return false; }
     virtual bool is_sharing_with_others() const override { return m_shared_with_others; }
+    void set_irq_shared_status(bool status) { m_shared_with_others = status; }
```

Comment:
`set_irq_shared_status` sounds like it sets something's shared *status*.
Let's be consistent with the internal member (`m_shared_with_others`) and call this `set_shared_with_others`)